firestore/aggregate_query_count_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package firestore
+
+import (
+	"bytes"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestCreateCountQuery(t *testing.T) {
+	projectID := os.Getenv("GOLANG_SAMPLES_FIRESTORE_PROJECT")
+	var bytes bytes.Buffer
+	err := createCountQuery(&bytes, projectID)
```

Comment:
Is this snippet dependent upon a collection already existing, or can it run on an empty project? If there are dependencies would be good to note.

Reply:
It is dependent on a query already existing. We have special projects just for Firestore (because you can't have Datastore & Firestore in the same project).

I've added a clarifying note.